core/pyos.py

```diff
@@ -313,6 +313,16 @@ def SignalState_AfterForkingChild():
   signal.signal(signal.SIGTTIN, signal.SIG_DFL)
 
 
+def SignalState_PendingTraps():
+    # type: () -> List[Tuple[int, _TrapHandler]]
+    return []
+
+
+def SignalState_ClearPendingTraps():
```

Comment:
A bit pedantic, but how about

- PendingTraps_Get()
- PendingTraps_Clear()

I think the `SignalState_` prefix was supposed for be for the "which signals the process responds to" state, and I think of pending traps as separate?

Maybe should rename that `SignalActions` or something